generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -101,4 +104,26 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
         }
     }
 
-}
\ No newline at end of file
+    protected compilePackageTdOptions(): object {
+        return {
+            "excludeNotExported": true,
+            "exclude": "lib, node_modules",
+            "experimentalDecorators": true,
+            "ignoreCompilerErrors": true,
+            "mode": "file",
+            "out": "./docs/" + this.getPackageName(),
+            "baseUrl": ".",
+            "paths": {
+                "@theia/*": ["node_modules/@theia/*"]
+            }
+        }
+    };
+
+    private getPackageName(): string {
+        let directoryName = `${process.cwd()}`;
```

Comment:
https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_cwd this already returns a string , no reason to use ` `... also if it were not a string you should use .toString()
Also use const rather than let